demo/molecule/autosuggestField/demo/index.js

```diff
@@ -82,6 +82,19 @@ const Demo = () => (
       />
     </div>
 
+    <div className={CLASS_DEMO_SECTION}>
+      <h3>With Alert validation HelpText</h3>
+
+      <MoleculeAutosuggestFieldWithState
+        id="with-error-validation-help-text"
```

Comment:
Is this id correct?
`with-**ERROR**-validation-help-text`

Reply:
Nope! I'm fixing it, thanks!